{"uuid": "1f252f65-585f-4b03-bfb1-355726d41146", "children": ["c60ee5b8-56c6-4655-8f8a-65fd14abc9e9"], "befores": [{"name": "browser", "status": "passed", "start": 1617621060623, "stop": 1617621060987}], "afters": [{"name": "browser::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\n\n", "trace": "  File \"D:\\Sonu\\personal\\xeneta-test\\python-page-object\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"D:\\Sonu\\personal\\xeneta-test\\python-page-object\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\Sonu\\personal\\xeneta-test\\python-page-object\\tests\\plugins\\browser.py\", line 10, in browser\n    browser.driver().close()\n  File \"D:\\Sonu\\personal\\xeneta-test\\python-page-object\\demoauto\\driver\\web_driver.py\", line 28, in close\n    self._driver.close()\n  File \"D:\\Sonu\\personal\\xeneta-test\\python-page-object\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 688, in close\n    self.execute(Command.CLOSE)\n  File \"D:\\Sonu\\personal\\xeneta-test\\python-page-object\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Sonu\\personal\\xeneta-test\\python-page-object\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1617621062788, "stop": 1617621062803}], "start": 1617621060623, "stop": 1617621062803}